src/Dialect/Krnl/KrnlOps.hpp

```diff
@@ -32,6 +32,7 @@ class KrnlOpsDialect : public Dialect {
       return LoopType::get(parser.getBuilder().getContext());
 
     parser.emitError(parser.getCurrentLocation(), "Unknown type");
+    return nullptr; 
```

Comment:
i think clang-format bot is complaining about the trailing white space.

Reply:
tx